src/controlPane.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+
+function startControlPane(pool, port) {
```

Comment:
as an alternative, we should export controlPlane or something that detects if the poolHall `isSupervisor` and sets up the control plane if it is.

The reason to do this is to allow the control plane to exist prior to `startPoolHall`, which will make it easier to attach worker/supervisor behavior similar to `heartbeat`